x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/plugin/ClusterRequestTests.java

```diff
@@ -165,7 +165,6 @@ public void testFallbackIndicesOptions() throws Exception {
         assertThat(cloned.clusterAlias(), equalTo(request.clusterAlias()));
         assertThat(cloned.sessionId(), equalTo(request.sessionId()));
         RemoteClusterPlan plan = cloned.remoteClusterPlan();
-        assertThat(plan.plan(), equalTo(request.remoteClusterPlan().plan()));
```

Comment:
Test was failing on this line because plans serialized with older transport versions are not equal to plans serialized with the new transport version.  This is expected, since when translating from the old version, we default the ts metadata to unknown, while the current version serializes and deserializes an actual value.

I discussed this failure with @dnhatn , and we concluded this assert wasn't adding any value in this test.